extensions/npm/package.json

```diff
@@ -112,6 +112,10 @@
       {
         "command": "npm.packageManager",
         "title": "%command.packageManager%"
+      },
+      {
+        "command": "npm.packageManagerLaunchArgs",
```

Comment:
@alexr00 thank you very much for taking a look at this so quickly! The plan with this command is as an enhancement or replacement of the existing `npm.packageManager` command, so that users of that command obtain command-line args that include the `nvm-exec` portion. Specifically the `js-debug` extension uses it, so I am happy to make a PR for `js-debug` to use it... I didn't want to double-down on my time spent before I knew if this PR would make the cut!